docs/crictl.md

```diff
@@ -154,11 +153,11 @@ container and `rm ID` to remove a container.
 ```console
 $ crictl version
 Version:  0.1.0
-RuntimeName:  cri-containerd
+RuntimeName:  containerd
 RuntimeVersion:  1.0.0-alpha.1-167-g737efe7-dirty
 RuntimeApiVersion:  0.0.0
 ```
-## Display Status & Configuration Information about Containerd & CRI-Containerd
+## Display Status & Configuration Information about Containerd & The CRI Plugin
 ```console
 $ crictl info
```

Comment:
The config is changed. We may need to generate a new output. At least add a TODO here.

Reply:
will put the todo on the issue..